Source/gui/SNTBinaryMessageWindowController.m

```diff
@@ -120,4 +141,40 @@ - (void)updateBlockNotification:(SNTStoredEvent *)event withBundleHash:(NSString
   });
 }
 
+#pragma mark - Standlone mode methods
```

Comment:
Could these two methods be pushed into the SwiftUI code?

It seems like `isAbleToAuthenticateInStandaloneMode:` is used to send an error into the UI but if that were in the UI we could more easily localize the error message itself. 

`approveBinaryForStandaloneMode` is a little less obvious but I think you could do the TouchID evaluation inside the SwiftUI code and then pass self.replyBlock() directly into the UI code instead of wrapping that in another callback?

Reply:
I can make that change. This is mostly left over from before we made the move to Swift UI. 

Part of this was also to make sure the window closed only after the request was processed. Hence the block wrapping back and forth.

Reply:
The current code pushes the local authorization checks into swift.